### PR TITLE
Added recipe for simple-salesforce

### PR DESCRIPTION
**Pull request by mariusvniekerk**
So this is using the newish python: noarch features since this is a pure python module.

Interested in working through how/if we can use some of these going forward instead of the traditional ones for python

### CODE REVIEW

**Comment by synapticarbors** [comment]
I don't think that `noarch` is currently supported by conda-forge, since it is a 4.3 feature and conda-forge still uses 4.2.

**Comment by mariusvniekerk** [comment]
@synapticarbors Ready for review

File: recipes/simple-salesforce/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+package:
+  name: simple-salesforce
+  version: "0.72.2"
+
+source:
+  fn: simple-salesforce-0.72.2.tar.gz
+  url: https://pypi.python.org/packages/fa/f6/84ac2f3cc92df3ab5debb25730e4f3bf65be0d62c5140e22b46eafaaed07/simple-salesforce-0.72.2.tar.gz
```
Comment (synapticarbors) [inline recipes/simple-salesforce/meta.yaml]: We prefer that the url should be replaced with the canonical form found in:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L18-L19

and the `md5` should be replaced with the `sha256` (`b8a4cc544282be1da41e1ef474c1cbf4368ebbc24f25df93e4bd84bf9a1bfea9`).

File: recipes/simple-salesforce/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  imports:
+    - simple_salesforce
+
+about:
+  home: https://github.com/simple-salesforce/simple-salesforce
+  description: 'A very simple Salesforce.com REST API client for Python'
+  summary: 'Simple Salesforce is a basic Salesforce.com REST API client. The goal is to provide a very low-level interface to the API, returning an ordered dictionary of the API JSON response.'
+  license: Apache-2.0
+  license_family: Apache
+  license_file: '{{ environ['RECIPE_DIR'] }}/LICENSE.txt'
```
Comment (synapticarbors) [inline recipes/simple-salesforce/meta.yaml]: add:

`dev_url: https://github.com/simple-salesforce/simple-salesforce`

**Comment by mariusvniekerk** [comment]
@synapticarbors changes made.

**Pull request merged**